fix.py

```
@@ -0,0 +1,128 @@
+with open("crates/logfwd-io/src/otap_receiver.rs", "r") as f:
```

Ruff (0.15.10)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)